modules/basic/module.nix

```diff
@@ -31,11 +31,12 @@ with builtins; {
 
     spellChecking = {
       enable = mkEnableOption "neovim's built-in spellchecking";
-      language = mkOption {
-        type = types.str;
-        description = "The language to be used for spellchecking";
-        default = "en_US";
-        example = "de";
+      enableProgrammingWordList = mkEnableOption "adds vim-dirtytalk, a wordlist for programmers, that includes programming words";
```

Comment:
in nmd (the docs builder) mkEnableOption appends "Enable" to the description

remove "adds" from the description here"